client/pages/stockItems.jsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
+import ErrorMessage from '../components/error-message';
+import LoadingSpinner from '../components/loading-spinner';
 
 export default function StockItems(props) {
 
+  if (props.state.error) {
+    <ErrorMessage msg="Connection error" />;
+  }
+
```

Comment:
Please be sure to test this to make sure that the message actually shows. You can update your state in the react dev tools to see if your ui re-renders properly.